Content.Server/IdentityManagement/IdentitySystem.cs

```diff
@@ -141,7 +141,7 @@ private string GetIdentityName(EntityUid target, IdentityRepresentation represen
         // Get their name and job from their ID for their presumed name.
         if (_idCard.TryFindIdCard(target, out var id))
         {
-            presumedName = id.FullName;
+            presumedName = id.FullName == string.Empty ? null : id.FullName;
```

Comment:
String.IsNullOrEmpty()?

Reply:
itll get set to null anyway if its null

Reply:
Maybe String.IsNullOrWhiteSpace
It might be better to add restrictions, and sanitation to the ID cards you can make though

Reply:
yea ill do isnullorwhitespace